pretyping/evarconv.mli

```diff
@@ -90,10 +90,16 @@ val activate_hook : name:CString.Map.key -> unit
 
 val apply_hooks : hook
 
-(** Check if a canonical structure is applicable *)
+(** Check if a canonical structure is applicable, raises [No_cs b] if the LHS is not an irreducible applied projection. b is true whenever the LHS is an applied projection, but reduces. *)
 
+exception No_cs of bool
```

Comment:
The only user of this function/exception ignores the bool.
I don't get the point of exposing the decompose_proj API. Could you please explain the choice?

Reply:
`decompose_proj` is used in `pretyping/unification.ml`.